hail/python/hailtop/utils/utils.py

```diff
@@ -108,6 +108,8 @@ def digits_needed(i: int):
 
 
 def grouped(n, ls):
+    if n < 1:
+        raise ValueError('invalid value for n')
```

Comment:
Let's also include the bad `n` value in the error message. You can use a [Python format string](https://docs.python.org/3/tutorial/inputoutput.html#fancier-output-formatting) to include the number `n` in the string itself.